CMakeLists.txt

```diff
@@ -5,8 +5,7 @@ include("common.cmake")
 # Haywire
 # ----------------------------------------
 project(haywire C)
-set(CMAKE_BUILD_TYPE RelWithDebInfo)
-#set(CMAKE_BUILD_TYPE Debug)
+set(CMAKE_BUILD_TYPE Release)
```

Comment:
I learned from @hyc that debug info is in its own segment of the file and never gets paged in during a regular run.